README.md

```diff
@@ -242,4 +257,29 @@ When using HTTP transport, the server will run on the configured port (default 8
 - MCP endpoint: `http://localhost:4200/mcp`
 - Health check: `http://localhost:4200/health`
 
+#### Using Per-Request Authentication with HTTP Transport
```

Comment:
"HTTP/SSE"

Reply:
Both HTTP and SSE are HTTP Transports (and SSE is being deprecated pretty hard). I think this is OK as-is